scripts/brackets.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+start Brackets.exe %*
```

Comment:
`start` is the command(just like open on Mac). The first argument to `start` is the program name. When this command is issued with the program name and optional arguments, it first searches for PATH variable, if it can find the program there. If not, it then goes through the registry entries present at `HKLM\SOFTWARE\Microsoft\Windows\Current Version\App Paths`. 

The change now would be that when the user types in `brackets`,  it is the command(`brackets.bat`) that is going to be executed and not the exe itself. This in turn uses `start` to launch `Brackets.exe` through the entry in the `App Paths` registry key.